backend/chat/commands/builtin/spam-raid-protection.js

```diff
@@ -2,6 +2,58 @@
 
 const chat = require("../../twitch-chat");
 const chatModerationManager = require("../../moderation/chat-moderation-manager");
+const commandManager = require("../CommandManager");
+const frontendCommunicator = require("../../../common/frontend-communicator");
+
+const activateProtectionOptions = async (commandOptions) => {
+    if (commandOptions.enableFollowerOnly) {
+        chat.enableFollowersOnly(commandOptions.enableFollowerOnlyDuration);
+    }
+
+    if (commandOptions.enableSubscriberOnly) {
+        chat.enableSubscribersOnly();
+    }
+
+    if (commandOptions.enableEmoteOnly) {
+        chat.enableEmoteOnly();
+    }
+
+    if (commandOptions.enableSlowMode) {
+        chat.enableSlowMode(commandOptions.enableSlowModeDelay);
+    }
+
+    if (commandOptions.clearChat) {
+        chat.clearChat();
+    }
+
+    if (commandOptions.banRaiders || commandOptions.blockRaiders) {
+        chatModerationManager.enableSpamRaidProtection(commandOptions.banRaiders, commandOptions.blockRaiders);
+    }
+
+    setTimeout(function() {
+        (function(commandOptions) {
+            chat.sendChatMessage(commandOptions.displayTemplate);
+        }(commandOptions));
```

Comment:
No need for the wrapping function.

Reply:
I don't know what I was thinking there? I must have been tired lol. Fixed!